packages/account/src/Components/error-component/error-component.tsx

```diff
@@ -29,8 +29,8 @@ const ErrorComponent = ({
                     : [localize('Sorry, an error occured while processing your request.'), refresh_message]
             }
             redirect_urls={[routes.trade]}
-            redirect_labels={[redirect_label || localize('Refresh')]}
-            buttonOnClick={redirectOnClick || (() => location.reload())}
+            redirect_labels={[redirect_label ?? localize('Refresh')]}
+            buttonOnClick={redirectOnClick ?? (() => location.reload())}
```

Comment:
same here, what is the sens of using here '??' as 'redirectOnClick' can be undefined or function, right?

Reply:
@yauheni-deriv, I think it's better to use `??` here, because we're checking for `undefined` or `null` and not a `boolean`  or a `number`, since this is a `function` definition

Reply:
It’s ok to leave it, but the reason to apply '??' is to not consider  0 or empty string as a falsy values. no our case I believe. :)